### ISSUE TITLE
Clients are gone

### ISSUE THREAD

**Issue by notevil1337** [UnamSanctam/SilentCryptoMiner#1868]
Bro a few of my clients disappeared after a few days of work that had windows defender. (I don't use rootkit)
Can you tell me what the rootkit is used for and whether you recommend enabling it for keep the clients for a long time ? @UnamSanctam

**Comment by UnamSanctam** [comment]
The rootkit is for hiding the miner from things like the Task Manager and similar things (things that run on ring 3), but antiviruses usually run on the kernel level (ring 0) so it wouldn't really make any difference in that case.

**Comment by UnamSanctam** [comment]
And Windows also recently released an update to Windows Defender that specifically targets my miner as well which could be the cause:
![detection](https://github.com/UnamSanctam/SilentCryptoMiner/assets/10323655/5c0a0517-c440-4749-9abd-81bb8382c1e5)

**Comment by notevil1337** [comment]
> And Windows also recently released an update to Windows Defender that specifically targets my miner as well which could be the cause: ![detection](https://user-images.githubusercontent.com/10323655/275322808-5c0a0517-c440-4749-9abd-81bb8382c1e5.png)

Hahaha yeah I saw that yesterday

**Comment by notevil1337** [comment]
I simply bypass WD by adding path to exclusions through my simple program and then drop the miner

**Comment by notevil1337** [comment]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141959915/33a3567e-bbb9-4f2a-967f-488d33eebfe8)
But some of them are gone (

**Comment by notevil1337** [comment]
If the path is added to the exclusions and the miner is already running, somehow the path was removed from the exclusions, will the antivirus find the miner after that?

**Comment by UnamSanctam** [comment]
Yes if the exclusion is removed it will search and remove the miner, though the watchdog will re-add the exclusions and reinstall the miner if the watchdog doesn't get killed (it usually doesn't). But if the exclusions get removed while the computer is starting/closing like what MSRT does then it will get fully removed since the watchdog can't be running then.

**Comment by notevil1337** [comment]
What would be the best thing to do to maintain the miner for a long time? Changing the code, using a crypter, rat or what ?
or it doesn't make sense and the antivirus will detect it

**Comment by svox07** [comment]
hi, I can crypt the file, it will take a long time to keep the miner

**Comment by notevil1337** [comment]
> hi, I can crypt the file, it will take a long time to keep the miner

what crypter do you use

**Comment by svox07** [comment]
this is my own crypter with a valid certificate

**Comment by notevil1337** [comment]
> this is my own crypter with a valid certificate
gimme discord

**Comment by notevil1337** [comment]
.. lmao

**Comment by i3eaa** [comment]
> this is my own crypter with a valid certificate

fr slide the dc

**Comment by svox07** [comment]
> > это мой собственный шифровщик с действительным сертификатом, 
> > дай мне дискорд

leave yours. Discord I will write to you

**Comment by notevil1337** [comment]
> > > это мой собственный шифровщик с действительным сертификатом,
> > > дай мне дискорд
> 
> leave yours. Discord I will write to you
scripter13

**Comment by i3eaa** [comment]
> > > это мой собственный шифровщик с действительным сертификатом,
> > > дай мне дискорд
> 
> leave yours. Discord I will write to you

incarnae

**Comment by svox07** [comment]
https://vimeo.com/837963397

**Comment by svox07** [comment]
https://vimeo.com/837963397

**Comment by svox07** [comment]
here is video evidence[tyts](https://vimeo.com/837963397)
